samtranslator/validator/sam_schema/definitions/intrinsics.json

```diff
@@ -0,0 +1,261 @@
+{
```

Comment:
Handcrafted but an experiment for now, we might not want it for now.
The `if` / `else` part is added to only report the errors from this specific definition for example.

Not sure we should validate Intrinsic functions for now, just now that something is an intrinsic (through the `intrinsic` type) and allow/disallow it.

Reply:
Got it. I think the thing we will need to be careful about is failing customers unexpectedly. The risk we will take in validating everything is if CFN updates or maybe a customer has some custom transform/custom resource that does something slightly differently than the defaults. Probably just overly cautious honestly, but just trying to think of ways this could hurt us.